privacy/rules.go

```diff
@@ -23,16 +27,18 @@ func (r ComponentEnforcementRule) Evaluate(target Component) ActivityResult {
 		return ActivityAbstain
 	}
 
+	if matched := evaluateGPPSID(r.gppSID, request); !matched {
+		return ActivityAbstain
```

Comment:
I noticed this return value isn't covered by tests. Just want to point it out.

Reply:
Test added.